### DISCUSSION TITLE
Windows NT and other things

### DISCUSSION THREAD

**Discussion by Alef15** [sqlpage/SQLPage#483] [General]
First of all, congratulations on a brilliant program!
It's simple, well done and I find the idea brilliant (obviously, it's exactly what I was looking for: a sort of ColdFusion from the 90s but which works here and now!).
I have a question: is there by any chance a version of SQLPage that could work with Windows 2000 (NT 5.0)?
I'm a fan of retrocomputing and I have an old NT server lying around at home... Well, I just read that SQLPage was developed with RUST and this language probably didn't exist at the time of Windows 2000 but It doesn't hurt to ask, does it?
Once again, congratulations for this idea and this achievement.
Regards.

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage !
  There have been successful attempts to compile modern rust code on old windows target. The most advanced is probably https://seri.tools/blog/announcing-rust9x/

  Let us know if you manage to compile SQLPage with that !
